docs/serving/autoscaling/README.md

```diff
@@ -0,0 +1,20 @@
+Knative Serving provides users with the functionality to automatically scale the number of application replicas based on demand.
+
+The component responsible for autoscaling functionality in Knative Serving is the **Autoscaler** component. The Autoscaler watches traffic flow to the application, and scales the application up or down based on metrics.
+
+The default Autoscaler used in Knative deployments is the Knative Pod Autoscaler (KPA). However, Knative deployments can be configured to use the Kubernetes Horizontal Pod Autoscaler (HPA) instead.
+<!-- Question: removed mention of BYO autoscaler since we don't seem to doc this yet - does it need to be added, can we provide instructions for it yet?-->
```

Comment:
@markusthoemmes just wondering if you know the status of BYO autoscaler and whether we want to doc using it yet?

Reply:
I don't think so. It's non-trivial and requires it's own set of development documentation.